src/macaron/slsa_analyzer/git_url.py

```diff
@@ -327,25 +327,11 @@ def clone_remote_repo(clone_dir: str, url: str) -> Repo | None:
     # Macaron attempting to clone a repository multiple times.
     # In these cases, we should not error since it may interrupt the analysis.
     if os.path.isdir(clone_dir):
-        logger.info("Checking if the repo %s needs cloning.", url)
         try:
             os.rmdir(clone_dir)
-            logger.info(
```

Comment:
Can you please change add back the log but use the debug level and log the `clone_dir` only. The log itself seems to be useful and good to keep when there is no secret in the parameters.

Reply:
Reverted in https://github.com/oracle/macaron/pull/352/commits/324de16baf5b72492a91c353a9c67fd4a92cbeac